projects/github-actions/test-results-to-slack/src/utils.js

```diff
@@ -39,10 +39,18 @@ async function getNotificationText( isFailure ) {
 		const { html_url, number, title } = github.context.payload.pull_request;
 		event = `PR <${ html_url }|${ number }: ${ title }>`;
 	}
-	if ( github.context.eventName === 'push' || github.context.eventName === 'schedule' ) {
+
```

Comment:
Do you think destructuring might make things a bit more readable here, maybe, and then easier to extend in the future? Something like 

```diff
diff --git a/projects/github-actions/test-results-to-slack/src/utils.js b/projects/github-actions/test-results-to-slack/src/utils.js
index afb9d5d1c2..dbfe90f34a 100644
--- a/projects/github-actions/test-results-to-slack/src/utils.js
+++ b/projects/github-actions/test-results-to-slack/src/utils.js
@@ -33,21 +33,23 @@ async function isWorkflowFailed( token ) {
  * @param {boolean} isFailure - whether the workflow is failed or not
  */
 async function getNotificationText( isFailure ) {
-	let event = github.context.sha;
+	const {
+		context: { eventName, sha, ref_type, ref_name, payload },
+	} = github;
 
-	if ( github.context.eventName === 'pull_request' ) {
-		const { html_url, number, title } = github.context.payload.pull_request;
+	let event = sha;
+
+	if ( eventName === 'pull_request' ) {
+		const { html_url, number, title } = payload.pull_request;
 		event = `PR <${ html_url }|${ number }: ${ title }>`;
 	}
 
-	if ( github.context.eventName === 'push' ) {
-		const { url, id } = github.context.payload.head_commit;
-		const { ref_type, ref_name } = github.context;
+	if ( eventName === 'push' ) {
+		const { url, id } = payload.head_commit;
 		event = `commit <${ url }|${ id }> on ${ ref_type } *${ ref_name }*`;
 	}
 
-	if ( github.context.eventName === 'schedule' ) {
-		const { ref_type, ref_name } = github.context;
+	if ( eventName === 'schedule' ) {
 		event = `scheduled run on ${ ref_type } *${ ref_name }*`;
 	}
 
```

Reply:
That's a good idea! Done with [977040d](https://github.com/Automattic/jetpack/pull/25651/commits/977040d9ed8a766fe7dbe2d85e9535640a4c4e98)